spec/features/usersignup_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
```

Comment:
Really good feature test 😀

Reply:
Thanks! we pul a lot of thought into it 🙂